support/android/apk/servoview/build.gradle

```diff
@@ -8,7 +8,6 @@ import java.util.regex.Pattern
 
 android {
     compileSdk 33
-    buildToolsVersion "33.0.2"
```

Comment:
This change breaks the build on NixOS. Although the `buildToolsVersion` is optional, [according to the documentation](https://developer.android.com/tools/releases/build-tools), it is not defaulted to the latest build tools for the SDK, but a version that the Android Gradle Plugin decides. 

```
You should always keep your Build Tools component updated by downloading the latest version using the [Android SDK Manager](https://developer.android.com/studio/intro/update). If you're using [Android plugin for Gradle 3.0.0](https://developer.android.com/studio/releases/gradle-plugin#3-0-0) or higher, your project automatically uses a default version of the build tools that the plugin specifies. To use a different version of the build tools, specify it using [buildToolsVersion](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.AppExtension.html#com.android.build.gradle.AppExtension:buildToolsVersion) in your module's build.gradle, as follows:
```
In my case, on NixOS, this means that Android Gradle Plugin will try to download and install the Build Tools version 30.0.3 into the *read-only* nix store and will fail.
```
Observed package id 'ndk;25.2.9519653' in inconsistent location '/nix/store/qdy9ddp1dahja6vrk2a3zjh3q807k4hp-androidsdk/libexec/android-sdk/ndk-bundle' (Expected '/nix/store/qdy9ddp1dahja6vrk2a3zjh3q807k4hp-androidsdk/libexec/android-sdk/ndk/25.2.9519653')
Observed package id 'ndk;25.2.9519653' in inconsistent location '/nix/store/qdy9ddp1dahja6vrk2a3zjh3q807k4hp-androidsdk/libexec/android-sdk/ndk-bundle' (Expected '/nix/store/qdy9ddp1dahja6vrk2a3zjh3q807k4hp-androidsdk/libexec/android-sdk/ndk/25.2.9519653')
Observed package id 'ndk;25.2.9519653' in inconsistent location '/nix/store/qdy9ddp1dahja6vrk2a3zjh3q807k4hp-androidsdk/libexec/android-sdk/ndk-bundle' (Expected '/nix/store/qdy9ddp1dahja6vrk2a3zjh3q807k4hp-androidsdk/libexec/android-sdk/ndk/25.2.9519653')
Errors during XML parse:
Additionally, the fallback loader failed to parse the XML.
Checking the license for package Android SDK Build-Tools 30.0.3 in /nix/store/qdy9ddp1dahja6vrk2a3zjh3q807k4hp-androidsdk/libexec/android-sdk/licenses
License for package Android SDK Build-Tools 30.0.3 accepted.
Preparing "Install Android SDK Build-Tools 30.0.3 (revision: 30.0.3)".
```

Reply:
Only that it is deprecated, will put it back.